dgraphtest/load.go

```diff
@@ -493,3 +493,28 @@ func (c *LocalCluster) BulkLoad(opts BulkOpts) error {
 		return nil
 	}
 }
+
+func (c *LocalCluster) CopyBulkLoadDirsToAlphaMounts() error {
```

Comment:
why are we doing the copy when we can just let the data stream?

Reply:
This is for the test when we want to copy `p` directories in each of the alphas in the cluster and start them. This is different from the case when we let the data stream from one alpha to all the other alpha (covered in: TestBulkLoaderSnapshotPDirinAlpha0)